src/components/form-control/form-control.jsx

```diff
@@ -37,7 +37,7 @@ export default function FormControl(props) {
         {props.children}
       </div>
       {isInvalid(props.error, props.readOnly) && (
-        <span className={styles['error-message']}>
+        <span id={`${props.id}Hint`} className={styles['error-message']} aria-live="polite" >
```

Comment:
Does the `id` do something in particular in this case? Usually, the ID is associated with another attribute: `label#for` or `element#aria-labelledby`, etc.

Reply:
So this is used with `aria-describedby`. If the input has `aria-invalid` and `aria-describedby` points to this span, the screen reader will read the error text when you have focus on the input. Pretty cool!

https://accessibility.huit.harvard.edu/provide-helpful-error-messages

eg.
```
<label for="email">Your email</label>
<input type="text" id="email" aria-invalid="true" aria-describedby="emailHint">
<div class="hint" id="emailHint" style="">Please enter a valid email.</div>
```

Reply:
Oh, I think I see a test that ensures this ID is part of some `aria-describedby`. I'll make this as resolved.